packages/astro/src/stores/external.ts

```diff
@@ -0,0 +1,31 @@
+import { computed } from 'nanostores';
+
+import { $clerk, $csrState, $initialState } from './internal';
+import { deriveState } from './utils';
+
+export const $authStore = computed([$csrState, $initialState], (state, initialState) => {
```

Comment:
Is this part of the public API? If so, can we add a JSDoc comment?

Reply:
Yes it is, i will add them